src/MainWindow.vala

```diff
@@ -58,9 +58,28 @@ public class MainWindow : Gtk.Window {
         web_context.set_process_model (WebKit.ProcessModel.MULTIPLE_SECONDARY_PROCESSES);
         web_context.get_cookie_manager ().set_accept_policy (WebKit.CookieAcceptPolicy.NO_THIRD_PARTY);
 
+        var web_kit_settings = new WebKit.Settings();
+        web_kit_settings.allow_file_access_from_file_urls = true;
+        web_kit_settings.allow_universal_access_from_file_urls = false;
+        web_kit_settings.default_font_family = Gtk.Settings.get_default().gtk_font_name;
+        web_kit_settings.enable_dns_prefetching = false;
```

Comment:
A lot of these are just setting the default values, maybe not ideal.

Reply:
@TingPing thanks, resolved in https://github.com/cassidyjames/ephemeral/commit/b6d04df55b5df97cddaf5c418deafd19f8de1399